python/python/lancedb/table.py

```diff
@@ -1188,11 +1189,25 @@ def create_index(
             index_cache_size=index_cache_size,
         )
 
-    def create_scalar_index(self, column: str, *, replace: bool = True):
+    def create_scalar_index(
```

Comment:
Does the API need to change for the remote table too?

Reply:
need to change the server side impl first, i think it doesn't respect the `index_type` param now cause we are passing `scalar` to it